src/v2/providers/alerts/appDistribution.ts

```diff
@@ -234,6 +265,79 @@ export function onNewTesterIosDevicePublished(
   return func;
 }
 
+/**
+ * Declares a function that can handle receiving new in-app feedback from a tester.
+ * @param handler - Event handler which is run every time new feedback is received.
+ * @returns A function that you can export and deploy.
+ */
+export function onInAppFeedbackPublished(
+  handler: (
+    event: AppDistributionEvent<InAppFeedbackPayload>
+  ) => any | Promise<any>
+): CloudFunction<AppDistributionEvent<InAppFeedbackPayload>>;
+
+/**
+ * Declares a function that can handle receiving new in-app feedback from a tester.
+ * @param appId - A specific application the handler will trigger on.
+ * @param handler - Event handler which is run every time new feedback is received.
+ * @returns A function that you can export and deploy.
+ */
+export function onInAppFeedbackPublished(
+  appId: string,
```

Comment:
Just to confirm, `appId` is optional when registering on Eventarc, right?

Reply:
Yes. It's the same as for `newTesterIosDevice`. Though I have to admit I didn't test this for either yet.

I guess we can test this as soon as http://cl/459802137 is rolled out to production. Do you have a good guesstimate when that might happen?

Reply:
Usually it takes about a week or so for a full rollout

Reply:
I was able to deploy successfully.